src/test/java/com/google/api/codegen/gapic/testdata/php/php_library.baseline

```diff
@@ -3103,7 +3103,7 @@ function sampleMonologAboutBook()
             $stream->write($request);
         }
         $response = $stream->readResponse();
-        printf("%s" . PHP_EOL, $response);
+        printf("The stage of the comment is: %s" . PHP_EOL, print_r($response->getStage()), true));
```

Comment:
Enums are represented as ints, see e.g. here: 

So we could just print them directly. I followup problem is that users are just going to get an int :-(

This PR was opened months ago to add string-to-int conversions and vice-versa https://github.com/protocolbuffers/protobuf/pull/4534 - seems like reviving it might be the best way to print something more useful.

Reply:
Awesome. Would it be OK to just print enums 'normally' for now and let's update the snippets when the pending PR is finished?

Reply:
@hzyi-google You're suggesting we print the ints for now, and then use the new protobuf functionality when https://github.com/protocolbuffers/protobuf/pull/4534 is merged?

If we see activity on that PR, then that's fine; let's not reinvent the wheel. If that doesn't seem to be moving along, then we might need to implement the functionality ourselves in the meantime.

Reply:
> But I guess in both cases we need to make efforts outside gapic-generator in order to print something meaningful

I'd love to track this ^ to make sure we follow-up 😄 

Up until recently, Python was "solving" this problem by creating a List of the enum display fields (from the proto) and printing via `list_of_display_names[response.enum_field]`. If the data model object snippets have access to includes a full representation of the proto, it's _technically_ possible for us to do the same thing (until https://github.com/protocolbuffers/protobuf/pull/4534 is merged). I'd love to see this help push the PR thru instead tho